windows/src/engine/keyman32/Keyman32.cpp

```diff
@@ -699,8 +699,9 @@ void HandleRefresh(int code, LONG tag)
     // of keyman32/keyman64 that a refresh is coming through
     Globals::PostMasterController(wm_keyman_refresh, KR_PRE_REFRESH, 0);
 
-    // We need to tell the controller windows to refresh themselves also
-    Globals::PostControllers(wm_keyman_control, KMC_REFRESH, 0);
+    // We need to tell the controller window to refresh itself also
+    // TODO: eliminate this message (we can hop onto the above message)
```

Comment:
I will leave this until a later date; it's not a real problem now; currently keyman.exe does not handle `wm_keyman_refresh` directly, only through keyman32.dll, so adding a handler for that does add complexity on the keyman.exe side of the equation.